notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino.ipynb

```diff
@@ -37,7 +37,7 @@
     "id": "KwQtSOz0VrVX"
```

Comment:
<p>please add some description and link on MO docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1001/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
Added